server/fleet/service.go

```diff
@@ -1059,6 +1059,12 @@ type Service interface {
 	// Returns empty status if the host is not a supported Linux host
 	LinuxHostDiskEncryptionStatus(ctx context.Context, host Host) (HostMDMDiskEncryption, error)
 
+	// GetLinuxDiskEncryptionSummary summarizes the current state of disk encryption for
+	// supported Linux hosts in the specified team (or, if no team is specified, each host that is not assigned to any team).
+
+	// TODO - custom return type, or just use relevant fields?
+	GetLinuxDiskEncryptionSummary(ctx context.Context, teamId *uint) (*MDMLinuxDiskEncryptionSummary, error)
```

Comment:
Probably use `MDMProfilesSummary` as the return type here, since that ensures we get the same format we're used to downstream for Win/Mac.